### ISSUE TITLE
JDK11 Windows : Failure in java_lang_ProcessBuilder_Basic/java_id0/Basic_id0

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9#10793]
Failure link
------------

https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.openjdk_x86-64_windows_Nightly/169/testReport/java_lang_ProcessBuilder_Basic/java_id0/Basic_id0/

For example, to rebuild the failed tests in <Jenkins URL>=https://ci.eclipse.org/openj9/job/Grinder, use the following links:
01:53:15  https://ci.eclipse.org/openj9/job/Grinder/parambuild/?JDK_VERSION=11&JDK_IMPL=openj9&BUILD_LIST=openjdk&PLATFORM=x86-64_windows&TARGET=jdk_lang_j9_0

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------
```
>'1'< not equal to '0'
java.lang.Throwable
	at java.base/java.lang.Thread.dumpStack(Thread.java:511)
	at Basic.fail(Basic.java:2747)
	at Basic.fail(Basic.java:2748)
	at Basic.equal(Basic.java:2754)
	at Basic.commandOutput(Basic.java:111)
	at Basic.commandOutput(Basic.java:122)
	at Basic.javaChildOutput(Basic.java:567)
	at Basic.pwdInChild(Basic.java:583)
	at Basic.realMain(Basic.java:1749)
	at Basic.main(Basic.java:2757)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.sun.javatest.regtest.agent.MainWrapper$MainThread.run(MainWrapper.java:127)
	at java.base/java.lang.Thread.run(Thread.java:836)
>'1'< not equal to '0'
java.lang.Throwable
	at java.base/java.lang.Thread.dumpStack(Thread.java:511)
	at Basic.fail(Basic.java:2747)
	at Basic.fail(Basic.java:2748)
	at Basic.equal(Basic.java:2754)
	at Basic.commandOutput(Basic.java:112)
	at Basic.commandOutput(Basic.java:122)
	at Basic.javaChildOutput(Basic.java:567)
	at Basic.pwdInChild(Basic.java:583)
	at Basic.realMain(Basic.java:1749)
	at Basic.main(Basic.java:2757)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.sun.javatest.regtest.agent.MainWrapper$MainThread.run(MainWrapper.java:127)
	at java.base/java.lang.Thread.run(Thread.java:836)
>''< not equal to 'C:\'
java.lang.Throwable
	at java.base/java.lang.Thread.dumpStack(Thread.java:511)
	at Basic.fail(Basic.java:2747)
	at Basic.fail(Basic.java:2748)
	at Basic.equal(Basic.java:2754)
	at Basic.realMain(Basic.java:1749)
	at Basic.main(Basic.java:2757)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.sun.javatest.regtest.agent.MainWrapper$MainThread.run(MainWrapper.java:127)
	at java.base/java.lang.Thread.run(Thread.java:836)
java.lang.AssertionError: Some tests failed
	at Basic.main(Basic.java:2759)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.sun.javatest.regtest.agent.MainWrapper$MainThread.run(MainWrapper.java:127)
	at java.base/java.lang.Thread.run(Thread.java:836)

JavaTest Message: Test threw exception: java.lang.AssertionError: Some tests failed
JavaTest Message: shutting down test

STATUS:Failed.`main' threw exception: java.lang.AssertionError: Some tests failed
```

**Comment by pshipton** [comment]
Hopefully resolved by https://github.com/eclipse/openj9/issues/10650

**Comment by JasonFengJ9** [comment]
> Hopefully resolved by #10650

It seems resolved in latest nightly builds, closing.
